laterpay/README.txt

```diff
@@ -91,7 +91,11 @@ the purchase button are displayed.
   WordPress plugin into your site on a test system, not on your production system. You can choose, whether LaterPay
   should be visible for your visitors in test mode or not.
 * Live mode: After integrating and testing the plugin, you might want to start selling content and process real
-  transactions. Please go to https://laterpay.net/form to sign up for a live merchant account. After the registration and the check of your contract data, we will send you your LaterPay API credentials for switching your plugin to live mode.
+  transactions. Please visit either the
+  (European Registration)[https://web.laterpay.net/dialog/entry/?redirect_to=/merchant/add#/signup] or
```

Comment:
Just to avoid confusion: Can we use
"If you you are based in the European Union:" / "If you are based in the USA:"
(that's consistent with what we have on the website, and people might not know what "region" refers to exactly in that context?